test/DotNetty.Transport.Tests/Channel/SingleThreadEventLoopTest.cs

```diff
@@ -470,10 +472,12 @@ public void TestOnEventLoopIteration()
         [Fact]
         public void TestRemoveOnEventLoopIteration()
         {
+            CountingRunnable onIteration2 = new CountingRunnable();
+#if DEBUG
             CountingRunnable onIteration1 = new CountingRunnable();
             _loopC.ExecuteAfterEventLoopIteration(onIteration1);
```

Comment:
This method is only available in debug so making it debug only as well but did not run them yet. Probably might need to skip for release as well